test/integration/compliance_history_test.go

```diff
@@ -443,6 +450,216 @@ var _ = Describe("GRC: [P1][Sev1][policy-grc] Test the compliance history API",
+				g.Expect(err).ToNot(HaveOccurred())
+
+				g.Expect(events).To(
+					HaveLen(expectedEvents),
+					fmt.Sprintf("expected cluster %s to have %d events", cluster, expectedEvents),
+				)
+
+				event0 := events[0].(map[string]interface{})
+				g.Expect(event0["event"].(map[string]interface{})["compliance"]).To(Equal("NonCompliant"))
+				g.Expect(event0["event"].(map[string]interface{})["message"]).
+					To(ContainSubstring("certificates expire in less than 30h0m0s"))
+				g.Expect(event0["event"].(map[string]interface{})["message"]).
+					To(ContainSubstring("certificates defined SAN entries do not match pattern Allowed: .*.test.com"))
+
+				event1 := events[1].(map[string]interface{})
+				g.Expect(event1["event"].(map[string]interface{})["compliance"]).To(Equal("Compliant"))
+				g.Expect(event1["event"].(map[string]interface{})["message"]).
+					To(ContainSubstring("All evaluated certificates are compliant"))
+
+			}
+		}, 120, 1).Should(Succeed())
+	})
+
+	It("Creates a policy with a noncompliant and compliant operator policy", func(ctx context.Context) {
+		now := time.Now().Add(-1 * time.Second).Format(time.RFC3339)
+		const opNs = "operator-policy-ns"
+		const opPolicyPath = "../resources/compliance_history/operator-policy-invalid.yaml"
+		const policyName = "op-compliance-api"
+
+		DeferCleanup(func(ctx context.Context) {
+			By("Deleting the operator policy")
+			_, err := common.OcHub(
+				"delete",
+				"-f",
+				opPolicyPath,
+				"-n", policyNS,
+				"--ignore-not-found",
+			)
+			Expect(err).ToNot(HaveOccurred())
+
+			// Delete subscription, catalog, etc at once
+			By("Deleting the namespace")
+			_, err = common.OcHub(
+				"delete",
+				"ns",
+				opNs,
+				"--ignore-not-found",
+			)
+			Expect(err).ToNot(HaveOccurred())
+		})
+
+		By("Creating the namespace")
+		_, err := common.OcHub(
+			"create",
+			"ns",
+			opNs,
+		)
+		Expect(err).ToNot(HaveOccurred())
+
+		By("Creating the Policy")
+		_, err = common.OcHub(
+			"apply",
+			"-f",
+			opPolicyPath,
+			"-n", policyNS,
+		)
+		Expect(err).ToNot(HaveOccurred())
+
+		clusters := verifyPolicyOnAllClusters(ctx, policyNS, policyName, "NonCompliant", defaultTimeoutSeconds)
+
+		By("Verifying that there are NonCompliant compliance events for the Operator parent policy")
+		Eventually(func(g Gomega) {
+			for _, cluster := range clusters {
+				events, err := listComplianceEvents(
+					ctx, "cluster.name="+cluster, "parent_policy.name="+policyName, "event.timestamp_after="+now)
+				g.Expect(err).ToNot(HaveOccurred())
+
+				event0 := events[0].(map[string]interface{})
+				g.Expect(event0["event"].(map[string]interface{})["compliance"]).To(Equal("NonCompliant"))
+				g.Expect(event0["event"].(map[string]interface{})["message"]).
+					To(ContainSubstring("CatalogSource 'invalid' was not found"))
+			}
+		}, 120, 1).Should(Succeed())
+
+		now = time.Now().Format(time.RFC3339)
```

Comment:
This helps with flakiness.
```suggestion
		now = time.Now().Format(time.RFC3339Nano)
```